airbyte-tests/src/automaticMigrationAcceptanceTest/java/io/airbyte/test/automaticMigrationAcceptance/MigrationAcceptanceTest.java

```diff
@@ -238,7 +239,11 @@ private static void assertDestinationDefinitionInformation(final ApiClient apiCl
           foundPostgresDestinationDefinition = true;
         }
         case "8be1cf83-fde1-477f-a4ad-318d23c9f3c6" -> {
-          assertEquals("0.2.0", destinationDefinitionRead.getDockerImageTag());
+          final String tag = destinationDefinitionRead.getDockerImageTag();
+          final AirbyteVersion currentVersion = new AirbyteVersion(tag);
+          final AirbyteVersion previousVersion = new AirbyteVersion("0.2.0");
+          final AirbyteVersion finalVersion = (currentVersion.checkOnlyPatchVersionIsUpdatedComparedTo(previousVersion) ? currentVersion : previousVersion);
+          assertEquals(finalVersion.toString(), currentVersion.toString());
```

Comment:
@tuliren this PR implement an auto-upgrade for patch versions, this means for migration tests that run 0.17/0.32/dev the LocalCSV was having some problems.
The version the static file is `0.2.0` and latest version of this connector is `0.2.9` so when running the `dev` migration this test was failing.

any suggestions or this solution is good?